docs/tasks/issuers/setup-acme/dns01/google.rst

```diff
@@ -35,7 +35,7 @@ To access this service account cert-manager uses a key stored in a Kubernetes Se
    gcloud iam service-accounts keys create key.json \
     --iam-account dns01-solver@$PROJECT_ID.iam.gserviceaccount.com
    kubectl create secret generic clouddns-dns01-solver-svc-acct \
-    --from-file=key.json
+    --from-file=key.json --namespace=cert-manager
```

Comment:
The secret resource should only be created in the `cert-manager` namespace if you are creating a *ClusterIssuer* resource. If you create an *Issuer* resource, it must be created in the same namespace as the issuer.

Could you maybe amend this PR to instead be a note explaining this?

You can use the note directive like so:

```
.. note:: Write your note here
```

Reply:
@munnerz Sorry, missed your suggestion.
I'm able to handle it in a week.

Reply:
@munnerz Is it good enough now. Do you think it makes sense to add the same note to other providers like azure...?